quake1_example/fluid.gd

```diff
@@ -1,6 +1,10 @@
 extends Area3D
 class_name QmapbspQuakeFluidVolume
 
+func _damage() -> int : return 0
+func _liquid_type() -> String : return 'water'
```

Comment:
```suggestion
func _liquid_type() -> StringName : return &'water'
```
I prefer to use StringNames over Strings for enum-like values. Because it described itself as a "name" and comparing them is faster than regular strings. For more information about them, see [here](https://docs.godotengine.org/en/stable/classes/class_stringname.html).